pkg/clusterdiscovery/clusterapi/clusterapi.go

```diff
@@ -15,19 +15,25 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog/v2"
 	clusterapiv1alpha4 "sigs.k8s.io/cluster-api/api/v1alpha4"
+	clusterapiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	secretutil "sigs.k8s.io/cluster-api/util/secret"
```

Comment:
Does this require us to update vendor versions?

Reply:
`sigs.k8s.io/cluster-api/api/v1beta1` requires

Reply:
Got it, thanks